src/utils/test_helper.rs

```diff
@@ -87,6 +87,13 @@ pub fn create_file_with_text<P: AsRef<Path>>(path: P, text: &str) -> () {
     file.write_all(text.as_bytes()).unwrap();
 }
 
+
+pub fn tag_lightweight<'repo>(repo: &'repo Repository, tag_name: &str) -> Reference<'repo> {
```

Comment:
what does lightweight mean?

Reply:
It's basically just a pointer vs an annotated tag with all kinds of other information. I feel like for the test I probably don't need to create a full on annotated tag, but am open to doing that instead if it makes more sense.

Reply:
If we already have signing set up for the integration tests we may as well?

Reply:
Ok, I accept this

Reply:
f687fb8